Analysis/Stan/update_yearly_dagar_timevary_speedup_flexible.stan

```diff
@@ -72,6 +73,15 @@ transformed data {
   real<lower=0> weights[M*(1-do_censoring)*use_weights]; //a function of the expected offset for each observation used to downwight the likelihood
   real log_meanrate = log(meanrate);
   real <lower=0> pop_loctimes[L]; // pre-computed population in each location period
+  real <lower=0, upper=1> tfrac_censoring[K1]; // tfrac accounting for censoring
```

Comment:
censored observations contribute "fully" to the likelihood but full observations might be scaled if set_tfrac = F so there are two separate sets of weights